index.html

```diff
@@ -0,0 +1,184 @@
+          Join our Webinar and learn how to protect your data <br>
+          In 2022
+        </h2>
+        <p class="p1">
+          As the world keeps evolving, it is importart for all members of the digital community to <br>
+          to learn how to protect there data from Cyber crime
+        </p>
+        <h2 class="date">
+          2022.10.18(TUE) ~20(THU)
+        </h2>
+        <p class="location">@Government Complex, Art Center</p>
+
+      </section>
+
+      <section class="main-program">
+        <h2 class="prg-header">
+          Main Program
+        </h2>
+        <hr class="orange-line">
+        <div class="black-area">
+
+          <div class="program">
+            <i class="fa-solid fa-user"></i>
+            <h3 class="prg-title">
+              Introduction
+            </h3>
+            <p class="prg-text">
+              Introduction to common Digital Attacks and cyber crime from 2015 till date
+            </p>
+          </div>
+          <div class="program">
+            <i class="fa-sharp fa-solid fa-database"></i>
+            <h3 class="prg-title">
+              Information Security
+            </h3>
+            <p class="prg-text">
+              Introduction to common Digital Attacks and cyber crime from 2015 till date
+            </p>
+          </div>
+          <div class="program">
+            <i class="fa-sharp fa-solid fa-database"></i>
+            <h3 class="prg-title">
+              Brokers & Big Data
+            </h3>
+            <p class="prg-text">
+              Introduction to common Digital Attacks and cyber crime from 2015 till date
+            </p>
+          </div>
+          <div class="program">
+            <i class="fa-solid fa-pen-nib"></i>
+            <h3 class="prg-title">
+              Network Security
+            </h3>
+            <p class="prg-text">
+              Introduction to common Digital Attacks and cyber crime from 2015 till date
+            </p>
+          </div>
+          <div class="program">
+            <i class="fa-solid fa-folder-open"></i>
+            <h3 class="prg-title">
+              Cyber Aware
+            </h3>
+            <p class="prg-text">
+              Introduction to common Digital Attacks and cyber crime from 2015 till date
+            </p>
+          </div>
+
+        </div>
+        <button class="join-btn">Join</button>
+        <h2 class="see-more">SEE THE WHOLE PROGRAM</h2>
+        <hr id="see-prg">
+      </section>
+
+      <!-- Featured Section -->
+      <section class="featured">
+
+        <h2 class="feature-header">Featured Speakers </h2>
+        <hr class="orange-line">
+
+        <div class="all-speakers">
+          <!-- column 1 -->
+          <div class="speaker1">
+
+
+
+          </div>
+
+          <!-- COlumn 2 -->
+          <div class="speaker2">
+
+          </div>
+        </div>
+        <div class="more-speakers">
+          <button class="more">MORE <i class="fa-solid fa-angle-down"></i></button>
```

Comment:
- [ ] Kindly fix the issue with the `see more` button because it's not functioning as it's supposed to.

![Animation2](https://user-images.githubusercontent.com/50721479/190703787-21ae04a7-d461-4884-b664-71752f8572f0.gif)

Reply:
this was an optional functionality. i will work on it later